### ISSUE TITLE
0009976:
E-Mails im Dateimanager ablegen / Drag&Drop an email to the Filemanager

### ISSUE THREAD

**Issue by Gloirin** [tine20/tine20#4987]
**Reported by gom on 11 Jun 2014 19:16**

**Version:** Collin (2013.10.7)

Zur Zeit ist es nicht möglich eine E-Mail im Dateimanager zu archivieren/abzulegen. Bsp. Durch Drag&amp;Drop auf den Dateimanager.

--------------------------

At this time it&#039;s not possible to drag and drop an email to Filemanager (to archive)

**Comment by Gloirin** [comment]
**Comment posted by htietze on 4 Jul 2014 09:34**

What is the use case to save complete emails in the file manager? For backup purpose you might move it to another mail folder, and attachments can be stored there right now (AFAIK).

**Comment by Gloirin** [comment]
**Comment posted by gom on 4 Jul 2014 10:06**

Man kann keine &quot;lokalen&quot; Verzeichnisse in dem E-Mail Client erstellen. Diese benötigen alle den Speicherplatz des IMAP-Kontos.

**Comment by Gloirin** [comment]
**Comment posted by htietze on 4 Jul 2014 10:25**

Well, that&#039;s a valid use case (if your IMAP account is not reliable). But as mspahn said in the forum, you could save the email locally and upload it to the file manager. That&#039;s very inconvenient. So I would rephrase the feature request to &quot;Provide local backup for emails&quot;. Under the hood it could be similar to the file manager but as a user I want emails in one place and files at another. Also it makes reactivation easier.

From the UI perspective I think about a special folder called backup with the option to create subfolders. Emails that are dropped into this folder gets archived. I would not show the archive folder in the file manager.

PS: English is the preferred language here because of the international audience. But continue to write in German if you want.

**Comment by Gloirin** [comment]
**Comment posted by gom on 11 Jul 2014 10:39**

The proposal it&#039;s okay. 
But I think global. Here a example: If you have a customer-folder, in this folder are all the documents, informations and emails. The emails should be open in tine20.

**Comment by pschuele** [comment]
we already implemented the "filing" of emails into the Filemanager/Mailfiler. closing this issue.
